src/index.js

```diff
@@ -1,92 +1,65 @@
-  listDivTitle.appendChild(listTitle);
-  listDiv.appendChild(listDivTitle);
-  const addTask = document.createElement('div');
-  addTask.className = 'add-task';
-  addTask.innerHTML = `
-  <input type="text" placeholder="Add a new task.." class="new-list-item add-item">
-  <img class="enter-key" src="${Enter}" alt="enter-key">
-  `;
-  listDiv.appendChild(addTask);
+const tasks = getStorage();
 
-  // Add list icon
-  const tickIcon = new Image();
-  tickIcon.src = Tick;
-  logoElement.appendChild(tickIcon);
-
-  // Add refresh icon
-  const refereshIconDiv = document.createElement('div');
-  refereshIconDiv.className = 'refresh-icon';
-  const refreshIcon = new Image();
-  refreshIcon.src = Refresh;
-  refereshIconDiv.appendChild(refreshIcon);
-  listDivTitle.appendChild(refereshIconDiv);
-
-  toDoList.forEach((list) => {
-    const listItem = document.createElement('div');
-    listItem.className = 'to-do-item';
-    listItem.innerHTML = `
-          <input id="done" type="checkbox" value="done">
-          <input type="text" value="${list.description}" class="new-list-item">
-          <img class="view-more" src="${ViewMore}" alt="view-more">
-    `;
-    listDiv.appendChild(listItem);
-  });
+// Delete an item from local storage
+const removeIndex = (index) => {
+  setStorage(Delete.deleteOne(getStorage(), index));
+  populateTasks(getStorage()); // eslint-disable-line
+};
 
-  const clearContent = document.createElement('div');
-  clearContent.className = 'clear-content';
-  clearContent.innerHTML = 'Clear all completed';
-  listDiv.appendChild(clearContent);
+const populateTasks = (arr) => {
+  currentTasks.innerHTML = '';
+  for (let i = 0; i <= arr.length; i += 1) {
+    // Add an item to local storage
+    const newDiv = document.createElement('div');
+    newDiv.className = 'to-do-item';
+    const tick = document.createElement('input');
+    const description = document.createElement('input');
+    description.className = 'task-description';
+    const menuImg = document.createElement('img');
+    menuImg.src = `${ViewMore}`;
+    tick.setAttribute('type', 'checkbox');
+    tick.id = `item${i}`;
+    newDiv.append(tick);
+    description.value = `${arr[i].description}`;
+    newDiv.append(description);
+    newDiv.append(menuImg);
+    currentTasks.appendChild(newDiv);
 
-  // Append elements to container
-  container.appendChild(logoElement);
-  container.appendChild(listDiv);
+    // Double click the input area to display the delete icon
+    description.addEventListener('dblclick', () => {
+      newDiv.classList.add('edit-mode');
+      newDiv.innerHTML = `<input type="checkbox" id="${i}"></input><input id = "update${i}" class="update" type="text" value = "${arr[i].description}"></input><i id="delete${i}" class="fas fa-trash-alt"></i>`;
+      document.getElementById(`update${i}`).focus();
+      document.getElementById(`delete${i}`).addEventListener('click', () => {
+        removeIndex(i);
+      });
```

Comment:
- [ ] the user can update the task now, however, when refreshing the page it changes back to the old value, this is due to the fact that the new value is not saved to the localStorage.

Reply:
resolved